### PR TITLE
MultiGPU GAN Colocation Issue.

### PR DESCRIPTION
**Pull request by Skylion007**
Fixed bug with collocation of gradients in Tensorflow 1.1. It won't colocate the gradients because it complains /GPU: is not the same as /device:GPU . Changing this line fixes the spurious warning messages when training on multiple GPUs.

### CODE REVIEW

**Comment by ppwwyyxx** [comment]
Opened an issue in tensorflow/tensorflow#11484.

**Comment by ppwwyyxx** [comment]
A (better) work around is easy so I also pushed one.
